acme/acme/client.py

```diff
@@ -136,7 +136,8 @@ def deactivate_authorization(self, authzr):
         """
         body = messages.UpdateAuthorization(status='deactivated')
         response = self._post(authzr.uri, body)
-        return self._authzr_from_response(response)
+        return self._authzr_from_response(response,
```

Comment:
It should have been in #7254 but wasn't caught. The defect wasn't found back then because the entire thing was dead code and the test coverage didn't check the URI. 

I discovered it when writing this PR because I needed to log any authz URIs that fail to deactivate.